internal/binutils/binutils_test.go

```diff
@@ -361,3 +362,31 @@ func TestLLVMSymbolizer(t *testing.T) {
 		}
 	}
 }
+
+func TestOpenMalformedELF(t *testing.T) {
+	// Test that opening a malformed ELF file will report an error containing
+	// the word "ELF".
+	bu := &Binutils{}
+	_, err := bu.Open(filepath.Join("testdata", "malformed_elf"), 0, 0, 0)
+	if err == nil {
+		t.Fatalf("Open malformed ELF succeeded but shouldn't have")
```

Comment:
Optional: for this and other error messages below: the common language for test failure messages in Go tests is "Foo: got X, want Y".